third_party/ruby/ruby-2.4.BUILD

```diff
@@ -603,10 +605,12 @@ void Init_ext(void)
     init(Init_date_core, "date_core");
     init(Init_bubblebabble, "digest/bubblebabble");
     init(Init_sha1, "digest/sha1");
-    init(Init_sha1, "digest/sha2.so");
+    init(Init_sha2, "digest/sha2.so");
```

Comment:
This is where `digest/sha2.so` was being registered with the contents of `digest/sha1.so`.